sources/Shaarei_Orah/Shaarei_Orah_parse.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+import cStringIO
+import pycurl
+from bs4 import BeautifulSoup
+# for a script located two directories below this file
+p = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, p)
+from sources.local_settings import *
+sys.path.insert(0, SEFARIA_PROJECT_PATH)
+os.environ['DJANGO_SETTINGS_MODULE'] = "local_settings"
+from sources.functions import *
+import re
+import codecs
+
+def url_to_soup(url):
```

Comment:
If you use the urllib2 library, this can be done in one line:
`soup = BeautifulSoup(urlib2.urlopen(url)).read(), 'html.parser')`